Project-1/Peihong_Xie_Project_1_50375781.md

```diff
@@ -0,0 +1,299 @@
+  (c) (A→(B∨C))∨(C→¬A) 
+  (d) ((A→B)∧C)∨(A∧D) 
+```
+
+Answer: 
+
+(a) Tautology
+| A | B	| C | ¬A→B | A∧¬C | (A∧¬C)→B | (¬A→B)∨((A∧¬C)→B) |
+| ---  | --- | --- | --- | --- | --- | --- |
+| 1 | 1 | 1 | 1 | 0 | 1 | 1 |
+| 1 | 1 | 0 | 1 | 1 | 1 | 1 |
+| 1 | 0 | 1 | 1 | 0 | 1 | 1 |
+| 0 | 1 | 1 | 1 | 0 | 1 | 1 |
+| 1 | 0 | 0 | 1 | 1 | 0 | 1 |
+| 0 | 1 | 0 | 1 | 0 | 1 | 1 |
+| 0 | 0 | 1 | 0 | 0 | 1 | 1 |
+| 0 | 0 | 0 | 0 | 0 | 1 | 1 |
+
+(b) Contingent
+| A | B	| A→B | A→¬B | (A→B)∧(A→¬B) | 
+| ---  | --- | --- | --- | --- |
+| 1 | 1 | 1 | 0 | 0 | 
+| 1 | 0 | 0 | 1 | 0 |
+| 0 | 1 | 1 | 1 | 1 |
+| 0 | 0 | 1 | 1 | 1 | 
+
+(c) Tautology
+| A | B	| C | B∨C | A→(B∨C) | C→¬A | (A→(B∨C))∨(C→¬A)) |
+| ---  | --- | --- | --- | --- | --- | --- |
+| 1 | 1 | 1 | 1 | 1 | 0 | 1 |
+| 1 | 1 | 0 | 1 | 1 | 1 | 1 |
+| 1 | 0 | 1 | 1 | 1 | 0 | 1 |
+| 0 | 1 | 1 | 1 | 1 | 1 | 1 |
+| 1 | 0 | 0 | 0 | 0 | 1 | 1 |
+| 0 | 1 | 0 | 1 | 1 | 1 | 1 |
+| 0 | 0 | 1 | 1 | 1 | 1 | 1 |
+| 0 | 0 | 0 | 0 | 1 | 1 | 1 |
+
+(d) Contingent
+| A | B	| C | D | A→B | (A→B)∧C | A∧D | ((A→B)∧C)∨(A∧D)
+| ---  | --- | --- | --- | --- | --- | --- |--- |
+| 1  | 1 | 1 | 1 | 1 | 1 | 1 | 1 |
+| 1  | 1 | 1 | 0 | 1 | 1 | 0 | 1 |
+| 1  | 1 | 0 | 1 | 1 | 0 | 1 | 1 |
+| 1  | 0 | 1 | 1 | 0 | 0 | 1 | 1 |
+| 0  | 1 | 1 | 1 | 1 | 1 | 0 | 1 |
+| 1  | 1 | 0 | 0 | 1 | 0 | 0 | 0 |
+| 1  | 0 | 1 | 0 | 0 | 0 | 0 | 0 |
+| 0  | 0 | 1 | 1 | 1 | 1 | 0 | 1 |
+| 0  | 1 | 1 | 0 | 1 | 1 | 0 | 1 |
+| 0  | 1 | 0 | 1 | 1 | 0 | 0 | 0 |
+| 1  | 0 | 0 | 1 | 0 | 0 | 1 | 1 |
+| 0  | 0 | 0 | 1 | 1 | 0 | 0 | 0 |
+| 0  | 0 | 1 | 0 | 1 | 1 | 0 | 1 |
+| 0  | 1 | 0 | 0 | 1 | 0 | 0 | 0 |
+| 1  | 0 | 0 | 0 | 0 | 0 | 0 | 0 |
+| 0  | 0 | 0 | 0 | 1 | 0 | 0 | 0 |
+
+[2] A _literal_ is an atomic formula or the negation of an atomic formula. We say a formula is in _conjunctive normal form_ (CNF) if it is the conjunction of the disjunction of literals. Find propositional logic formulas in CNF equivalent to each of the following:
+  ```(a) (A→B)→C
+  (b) (A→(B∨C))∨(C→¬A)
+  (c) (¬A∧¬B∧C)∨(¬A∧¬C)∨(B∧C)∨A 
+```
+Answer:
+
+(a) CNF: (A∨C)∧(¬B∨C)
+
+(b) CNF: B∨¬B (the CNF of a tautology is any formula of the form "p∨¬p")
+
+(c) CNF: B∨¬B (the CNF of a tautology is any formula of the form "p∨¬p")
+
+[3] Let V be the vocabulary of first-order logic consisting of a binary relation P and a unary relation F. Interpret P(x,y) as “x is a parent of y” and F(x) as “x is female.” Where possible define the following formulas in this vocabulary; where not possible, explain why: 
+  ```(a)  B(x,y) that says that x is a brother of y
+  (b)  A(x,y) that says that x is an aunt of y
+  (c)  C(x,y) that says that x and y are cousins 
+  (d)  O(x) that says that x is an only child
+  (e)  T(x) that says that x has exactly two brothers 
+```
+
+Answer:
+
+
+(a) B(x,y) $\leftrightarrow$ ∃u(P(u,x)∧P(u,y))∧¬F(x)∧¬(x=y)
+
+(b) A(x,y) $\leftrightarrow$ ∃u∃w(P(u,y)∧P(w,x)∧P(w,u)∧¬(x=u))∧F(x)
+
+(c) C(x,y) $\leftrightarrow$ ∃u∃w∃z(P(w,z)∧P(w,u)∧P(z,x)∧P(u,y)∧¬(z=u))∧¬(x=y)
```

Comment:
Counterexample: Second cousins

Reply:
> Second cousins

I revise my formula to adapt to the possibility of second cousins. But I guess that we can have third or fourth or ... cousin by appealing to great-great-grandparent or great-great-great-grandparent or ... In this way, such a formula can be constructed to be infinitely long. Just as Ali said, maybe all people are cousins on this giant piece of rock.